Jenkinsfile

```diff
@@ -9,6 +9,11 @@ def gitEnv = [
 
 env.CHANGE_BRANCH = env.CHANGE_BRANCH ?: env.BRANCH_NAME
 
+properties([
+    // Run every hour
```

Comment:
Running Jenkins job every hour, isn't that excessive use of resources? Ideally, we should use GitHub's HTTP hooks to trigger the run of this job right after loopback-next's master branch was changed, or perhaps use some sort of npm mechanism to run the job after a new version of `@loopback/docs` was published.

If there isn't any easy solution for triggering the run only when there are changes to pull, then I am proposing to change this frequence to something like once a day, or at least every 4 hours or so.

cc @rmg

Reply:
Yeah I had a discussion with @rmg and running every hour isn't excessive from what he told me (correct me if I'm wrong, @rmg). But I would like us to use Travis instead of Jenkins for updating the docs, so this is probably outdated (I'll probably revert/disable this once we have Travis up and running).